README.rst

```diff
@@ -154,7 +154,11 @@ see ``docs/suppport.rst`` in the source distribution).
 
 .. start-badges
 
-|doc| |build| |coverage| |bestpractices|
+|bounties| |doc| |build| |coverage| |bestpractices|
```

Comment:
Sure.  I noticed the color of the bounty source badge is different from the others - I'm not sure how or if this can be changed